marin/speedrun/create_leaderboard.py

```diff
@@ -41,24 +45,32 @@ def create_entry_from_results(results: dict, results_filepath: str) -> Leaderboa
     filepath = Path(results_filepath)
     repo_root = Path(__file__).resolve().parent.parent.parent
     relative_path = filepath.relative_to(repo_root).parent
-    total_training_flops = results["run_stats"]["total_training_flops"]
-    training_time = results["run_stats"]["training_time_in_minutes"]
-    eval_paloma_c4_en_bpb = results["run_stats"]["eval/paloma/c4_en/bpb"]
-    model_size = results["run_related_info"]["num_parameters"]
-    submitted_by = results["run_related_info"].get("submitted_by", "unknown")
-    wandb_link = results["run_related_info"].get("wandb_run_link", None)
-    run_timestamp = results["run_related_info"].get("run_completion_timestamp", None)
+    total_training_flops = results["run_info"]["total_training_flops"]
```

Comment:
specify that this is hardware flops?  how about `training_hardware_flops` as opposed to `training_model_flops`?